RFXtrx/lowlevel.py

```diff
@@ -2693,6 +2693,39 @@ def _set_strings(self):
                 self.cmnd_string = self._UNKNOWN_CMND.format(self.cmnd)
 
 
+###############################################################################
+# DSMR class
+###############################################################################
+
+class Dsmr(SensorPacket):
+    """
+    Data class for the Dsmr packet type
+    """
+    TYPES = {0x01: 'P1',
+             0x02: 'Teleinfo'}
+
+    def load_receive(self, data):
+        """Load data from a bytearray"""
+        self.data = data
+        self.packetlength = data[0]
+        self.packettype = data[1]
+        self.subtype = data[2]
+        self.seqnbr = data[3]
+        self.dsmr_data = bytes(data[4:-2]).decode('ascii')
+        self._set_strings()
+
+    def _set_strings(self):
+        """Translate loaded numeric values into convenience strings"""
+        self.id_string = "dsmr:{0}".format(self.subtype)
```

Comment:
This might be why you get issues in home assistant . I'd string is expected to be fully numeric, possible with  a pair of numbers separated by :